ui/app/src/components/PathNavigator/PathNavigatorItem.tsx

```diff
@@ -33,6 +33,7 @@ export interface NavItemProps {
   locale: string;
   isLeaf?: boolean;
   isActive?: boolean;
+  isRoot?: boolean;
```

Comment:
What we're adding is the current path's item, not the root path item. We use root for the item/path the widget is rooted at (i.e. rootPath). Let's rename `isRoot` to `isCurrentPath`.